doc/user_guide/dynamo0p3.rst

```diff
@@ -744,6 +745,12 @@ later in this section (see :ref:`dynamo0.3-kernel-valid-access`).
   face. This means that such a Kernel needs appropriate
   synchronisation (or colouring) to run in parallel.
 
+* ``GH_READINC`` indicates that the data is first read and then
+  subsequently incremented. Therefore this is equivalent to a
+  ``GH_READ`` followed by a ``GH_INC``. It is important that
```

Comment:
I agree that there should be information but I think it should be more about correctness in the user guide and more about implementation in the developer guide. I've added some words for your enjoyment in any case. Note, most of the words are actually about not adding a halo exchange for gh_inc, rather than adding one for gh_readinc and the potential ensuing problems.

Reply:
Thanks for adding the clarifications 👍